db/event_manager_test.go

```diff
@@ -737,9 +767,11 @@ func TestWebhookTimeout(t *testing.T) {
 			errCount++
 		}
 	}
-	time.Sleep(15 * time.Second)
 	// Even though we timed out waiting for response on the SG side, POST still completed on target side.
-	assert.Equal(t, 10, wr.GetCount())
+	err = em.waitForProcessedTotal(context.TODO(), 10, 30*time.Second)
```

Comment:
As discussed, looks like there's a race between the webhook timeout (1s) and the queue timeout (1.1s).  Should be able to avoid by reducing the webhook timeout to 500ms and increasing the queue timeout to 1500ms.  

Should also add additional comments in these tests to make it clear that 'timeout' refers to the webhook timeout (set on line 758), and 'waitForProcess' is the eventManager waitTime.